R/assertions.R

```diff
@@ -1658,20 +1661,30 @@ assert_date_vector <- function(arg, optional = FALSE) {
 #'   false_value = 0,
 #'   missing_value = "missing"
 #' ))
-assert_same_type <- function(...) {
+assert_same_type <- function(...,
+                             .message = c(
+                               "Arguments {.arg {arg_names}} must be the same type.",
```

Comment:
Should it be "must be of the same type"?

Reply:
I think either phrasing works well. I prefer it as it is because it's just a bit shorter.